BootloaderCorePkg/Tools/SingleSign.py

```diff
@@ -94,6 +93,16 @@ def run_process (arg_list, print_cmd = False, capture_out = False):
 
     return output
 
+def get_key_sz_type ():

```

Comment:
Do we really need this ?   
Can't this be determined from the  key itself ?

Reply:
We have defined generic key id's as MASTER_KEY_ID. Ideally, signing interface (single singing) should decide key type to used for signing. Currently we support all platforms with same interface script. Legacy platforms use RSA2K and some platforms need RSA3K.
We would need to define a method to intimate signing interface script which key type to be used as RSA2048, RSA3072, ECC or so. Also, we do not wann to maintain different key id's for various key types.

Understand adding a new Env variable increase number of of variable settings. Looking for recommendations here.

Reply:
Yes, if we use actual key file this can be handled currently.